README.md

```diff
@@ -122,6 +122,21 @@ Your workflow should look something like this:
 
 Alternatively, you can clone/download the Alfred-Workflow [repository][repo] and copy the `workflow` subdirectory to your workflow's root directory.
 
+<a name="python-2-support-on-macos-123"></a>
+### Python 2 support on macOS 12.3+ ###
+
+Python 2 was [removed in macOS 12.3](https://developer.apple.com/documentation/macos-release-notes/macos-12_3-release-notes#:~:text=Python%202.7%20was%20removed%20from%20macOS%20in%20this%20update). Users that run macOS 12.3 or newer will have to [install Python 2.7.18 manually](https://www.python.org/downloads/release/python-2718/).
+Additionally, the following code needs to be added to the `info.plist` file after `<plist><dict>`:
+
+```xml
+<key>variables</key>
+<dict>
+    <key>PATH</key>
+    <string>/usr/local/bin:/usr/bin:/Library/Frameworks/Python.framework/Versions/2.7/bin</string>
```

Comment:
[Same issue as the other PR](https://github.com/danielecook/gist-alfred/pull/18#discussion_r878119038). Also, you’re not including important path segments, such as `/usr/sbin` and `/bin`. Finally, there’s no need for `/Library/Frameworks/Python.framework/Versions/2.7/bin`—if the user is on a version which has that, they’ll also have `/usr/bin/python`.

Reply:
@vitorgalvao 

I installed Python 2 from the [official website](https://www.python.org/downloads/release/python-2718/) and here is how it looks like here.

```
❯ which /usr/bin/python
/usr/bin/python not found

❯ which /opt/homebrew/bin/python
/opt/homebrew/bin/python not found

❯ which python
/Library/Frameworks/Python.framework/Versions/2.7/bin/python
```

Reply:
> I installed Python 2 from the [official website](https://www.python.org/downloads/release/python-2718/)

That’s not the recommended way to do it, Alfred-wise[^1]. But it is *a* solution, so no harm in having `/Library/Frameworks/Python.framework/Versions/2.7/bin` for those who followed your steps.

> Additionally, does any Python 2 installer actually populate `/usr/sbin` and `/bin`?

No, but Workflows are often more than one language and editing `PATH` in the Workflow Environment Variables will affect every object. So if the Workflow is mostly Python but actioning a result does some quick thing in another language, it might break due to a missing system binary. It’s a matter of not removing default locations developers take for granted; adding those to `PATH` has no cost, but removing them might.

[^1]: Installing from PKG is harder to completely get rid of, and because it installs outside the home directory it can—though is unlikely to—affect other stuff.

Reply:
iirc `PATH` is undefined by default in Alfred workflows, and it doesn't inherit variables from any rc/profile shell files.

How not adding these extra locations can break workflows since they are already not there to begin with?

Reply:
> iirc `PATH` is undefined by default in Alfred workflows

If `PATH` weren’t defined, every Workflow would need to use full binary paths for every standard tool. [`PATH` *is* defined in Alfred](https://www.alfredapp.com/help/workflows/advanced/understanding-scripting-environment/) as `/usr/bin:/bin:/usr/sbin:/sbin`.

> it doesn't inherit variables from any rc/profile shell files.

That is correct, but not inheriting from the user’s profile is not the same as not having environment variables set. Alfred still has them, which is why you can still (for example) check `$HOME` inside Alfred and it will give you the correct output.

> How not adding these extra locations can break workflows

By setting a new `PATH` which *removes* locations Alfred has by default, a Workflow which was working can break. Naturally, if it will happen depends on the Workflow in question.

Reply:
@vitorgalvao Done, thank you.